internal/mode/static/telemetry/collector_test.go

```diff
@@ -379,14 +405,43 @@ var _ = Describe("Collector", Ordered, func() {
 					RouteAttachedClientSettingsPolicyCount:   2,
 					ObservabilityPolicyCount:                 1,
 					NginxProxyCount:                          1,
+					SnippetsFilterCount:                      3,
 				}
 				expData.ClusterVersion = "1.29.2"
 				expData.ClusterPlatform = "kind"
 
-				data, err := dataCollector.Collect(ctx)
+				expData.SnippetsFiltersContextDirectives = []string{
+					"http-aio",
+					"location-keepalive_time",
+					"main-worker_priority",
+					"server-auth_delay",
+					"http-client_body_timeout",
+					"location-allow",
+					"main-worker_rlimit_nofile",
+					"server-ignore_invalid_headers",
+				}
+				expData.SnippetsFiltersContextDirectivesCount = []int64{
+					2,
+					2,
+					2,
+					2,
+					1,
+					1,
+					1,
+					1,
+				}
 
+				data, err := dataCollector.Collect(ctx)
 				Expect(err).ToNot(HaveOccurred())
-				Expect(expData).To(Equal(data))
```

Comment:
Was this not working anymore?

Reply:
oops, in some previous iterations I had to remove this since I had changed to checking the lists with `ContainsElements` then `ConsistsOf`, but after ordering both lists I changed them back to `Equal` so all the stuff I added should be removed and just checking on `expData` to `data` should work